clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/deploy/DescriptionValidator.groovy

```diff
@@ -33,40 +33,4 @@ public abstract class DescriptionValidator<T> implements VersionedCloudProviderO
   }
 
   abstract void validate(List priorDescriptions, T description, Errors errors)
-
-  @Autowired(required = false)
-  FiatPermissionEvaluator permissionEvaluator
-
-  void authorize(T description, Errors errors) {
```

Comment:
Discovered there are a handful (or more!) descriptions that do not implement `DescriptionValidator` and thus are ignored as far as authorization checks go.

I don't think `implements DescriptionValidator` should be a pre-req and this logic has been extracted / converted to Java.